Lib/fontbakery/commands/check_specification.py

```diff
@@ -192,6 +192,9 @@ def main(specification=None, values=None):
         values_[key] = getattr(args, key)
 
   try:
+    if not args.fonts:
+      raise ValueValidationError, "No files to check specified."
```

Comment:
Hmm, I dislike this for two reasons:
1. now a spec has to have a font's argument, `check-specification` was clean of that before and specifications could define their dependencies much more freely. 
2. There's a mechanism that raises exactly this error and ends the script (with this PR also with `sys.exit(1)` So why do it two times?

Reply:
fixed in #1777